README.md

```diff
@@ -296,10 +296,13 @@ dependencies in the Xargo.toml file as you would do with Cargo.toml:
 [utest]: https://github.com/japaric/utest
 
 ``` toml
+# First build some standard crates.
 [dependencies.alloc]
 [dependencies.panic_abort]
 [dependencies.panic_unwind]
 
+# Then build our custom facade. This depends on the crates
```

Comment:
Instead of saying "This depends on the crates above", maybe instead say "This implicitly depends on the above crates being available in the sysroot".

Reply:
Like so?

Reply:
:+1: looks good.